src/helpers/_shade.scss

```
@@ -0,0 +1,8 @@
+/// Slightly darken a color
+/// @access public
+/// @param {Color} $color - color to shade
```

We _generally_ use colour reasonably consistently throughout, unless we're directly referencing something from CSS itself (`currentColor`, the color property).

```suggestion
/// @param {Colour} $colour - colour to shade
```